src/ast/nodes/CallExpression.ts

```diff
@@ -66,6 +67,9 @@ export default class CallExpression extends NodeBase implements DeoptimizableEnt
 				this
 			);
 		}
+		if (this.callee instanceof MemberExpression && !this.callee.variable) {
```

Comment:
Funny - I had this initially but dismissed it due the higher number of test regressions. But it's the safer route.

Reply:
Well, if we gonna fix it, we might just as well fix it all the way down. I kept the tests because I REALLY hope to get in a proper solution soon.